internal/testutil/env.go

```diff
@@ -0,0 +1,23 @@
+package testutil
+
+import (
+	"os"
+)
+
+// SetEnvIfUnset sets the environment variable with the given name to the specified value
+// if it is not already set. It returns true if the environment variable was not previously set,
+// and false if it was already set.
+//
+// Parameters:
+//   - name: The name of the environment variable.
+//   - value: The value to set the environment variable to if it is not already set.
+//
+// Returns:
+//   - bool: True if the environment variable was not previously set, false otherwise.
+func SetEnvIfUnset(name, value string) bool {
+	_, found := os.LookupEnv(name)
+	if !found {
+		os.Setenv(name, value)
+	}
+	return !found
+}
```

Comment:
I'd rather we make this more opinionated - I don't think there's any reason each test should have to pass in the name and value, since those will always be the same. Would be better to be able to invoke the function without any arguments